src/openzaak/components/zaken/openapi.yaml

```diff
@@ -3597,20 +3596,20 @@ components:
           type: string
           maxLength: 40
           minLength: 1
-        subAdresBuitenland_1:
+        subAdresBuitenland1:
```

Comment:
the underscore-to-camelCase thing is doing something odd here, this *may* be configurable.